util/parseArchDB.py

```diff
@@ -1,8 +1,7 @@
-#!/usr/bin/python
```

Comment:
I think, you can refactor package names too with this [convention](https://www.python.org/dev/peps/pep-0008/#package-and-module-names).

Reply:
Converted script's names to lowercase.